src/main/java/net/minecraftforge/fml/common/registry/FMLControlledNamespacedRegistry.java

```diff
@@ -935,4 +938,15 @@ public void remove()
         return new RegistryEvent.Register<I>(location, this);
     }
 
+    @Override
+    public void writeEntry(ByteBuf out, I entry)
```

Comment:
The nullability of Forge registries is ambiguous. Minecraft uses AIR as default values for its registries, but Forge's can return null or a default value.